### DISCUSSION TITLE
盟军敢死队2  1.20汉化版

### DISCUSSION THREAD

**Discussion by 972933728** [Gamuxorg/bbs#389] [投递和请求游戏]
如题

**Comment by 972933728** [comment]
求大神帮忙打包一下盟军敢死队2  1.20版汉化版，这个游戏是可以用wine玩，但是只能英文，英文版还没有字幕。盟军敢死队2的汉化补丁不是那种覆盖文件替换文本的，而是一个新的程序来运行游戏的。
盟军敢死队2 1.20汉化版基本灭绝了，我给个1.20汉化版的链接
https://pan.baidu.com/s/1ud7Z-yY-Rc0jtg0Yra79sA 提取码: pppm

**Comment by 972933728** [comment]
GSooRun.exe是启动汉化版的程序，这个玩意无法用wine启动，属于外挂汉化的那种。

**Comment by wangling12** [comment]
wine游戏就不打包了
解决方法：
安装winetricks
然后执行：
winetricks dlls vcrun6 
在我这边就可以直接运行了，如果有问题请附上log
![2019-06-05 23-14-52 的屏幕截图](https://user-images.githubusercontent.com/24770600/58967969-c7c0b080-87e7-11e9-93eb-95bf28a0a999.png)

**Comment by 972933728** [comment]
你能不能给我一份杀手4血钱1.2版的
你打包的那个每次都要重新启动电脑才能玩
给我一份1.2版 我用steam play玩，我找不到1.2版的，只有1.0的BUG版本会自动推出。





------------------ 原始邮件 ------------------
发件人: "wangling12"<notifications@github.com>;
发送时间: 2019年6月5日(星期三) 晚上11:16
收件人: "Gamuxorg/bbs"<bbs@noreply.github.com>;
抄送: "fox"<aaa972933728@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [Gamuxorg/bbs] 盟军敢死队2  1.20汉化版 (#51)




wine游戏就不打包了
 解决方法：
 安装winetricks
 然后执行：
 winetricks dlls vcrun6
 在我这边就可以直接运行了，如果有问题请附上log

**Comment by wangling12** [comment]
需要重启才能打开是因为上次没有完全退出，你可以从终端运行appimage，退出游戏后，在终端ctrl+c强制结束脚本。
安装包我删了，你可以去rutracker.org找种子，自己下载
